checks.d/etcd.py

```diff
@@ -112,4 +112,5 @@ def get_json(self, url, timeout):
                 tags = ["url:{}".format(url)])
             return None
 
+        self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK, tags=["url:{}".format(url)])
```

Comment:
@remh: indeed, fixed now.
@LeoCavaille: also fixed. (and thanks for the explanation)